mailbox/api/src/main/java/org/apache/james/mailbox/model/UpdatedFlags.java

```diff
@@ -104,7 +143,19 @@ private static boolean isChanged(Flags original, Flags updated, String userFlag)
     public Flags getOldFlags() {
         return oldFlags;
     }
-    
+
+    public boolean isSet(Flags.Flag flag) {
```

Comment:
it's not a flags update, it's an UpdatedFlags.
There's something wrong in the mix of present and past, I guess.